app/ember/src/client/preview/index.js

```diff
@@ -3,7 +3,7 @@ import { start } from '@storybook/core/client';
 import './globals';
 import render from './render';
 
-const { load: coreLoad, clientApi, configApi, forceReRender } = start(render);
+const { configure: coreConfigure, clientApi, forceReRender } = start(render);
```

Comment:
Is due to the api is typically shaped as `configure` so we need to rename it here?  I think when someone read the code and it would be quite surprise that there are subtle differences. Or what `start` returned can just return the object with the preferred name, i.e. `coreConfigure`?!  (that wasn't too problematic though, just some naive feedback)

Reply:
Yeah this will need to be refactored. But it would be a a breaking change, so that's 6.0 material (and may be superceded by monoconfig anyway?O

Reply:
Right, just leave it like that.  And does any plan on 6.0 started (cc @ndelangen)?